packages/core/src/components/button/buttons.tsx

```diff
@@ -73,69 +72,17 @@ AnchorButton.displayName = `${DISPLAYNAME_PREFIX}.AnchorButton`;
  */
 function useSharedButtonAttributes<E extends HTMLAnchorElement | HTMLButtonElement>(
     props: E extends HTMLAnchorElement ? AnchorButtonProps : ButtonProps,
-    ref: React.Ref<E>,
+    passedRef: React.Ref<E>,
```

Comment:
Why are we renaming this?

Reply:
Oops, left over from a previous iteration, no longer necessary.